go/darwin/Makefile

```
@@ -1 +1,8 @@
 include ../Makefile.common
+
+ifeq ($(DOCKER_MULTIARCH),1)
+DOCKER_CMD := $(SELF_DIR)/../.ci/scripts/buildx.sh
+
+push:
+	@echo "Already done."
```

```suggestion
```suggestion
	@echo "Already done by buildx (.ci/scripts/buildx.sh)."
atomic-push:
	@echo "Already done by buildx (.ci/scripts/buildx.sh)."
```
```